helix-rest/src/test/java/org/apache/helix/rest/metadatastore/accessor/TestZkRoutingDataReader.java

```diff
@@ -42,12 +42,16 @@
 
   @BeforeClass
   public void beforeClass() {
+    ZK_SERVER_MAP.get(ZK_ADDR).getZkClient()
+        .deleteRecursively(MetadataStoreRoutingConstants.ROUTING_DATA_PATH);
```

Comment:
Nit: a good addition to this would be a TestHelper.verify() that checks that the deletion has completed.

Reply:
This should be a must because I believe there would be a latency for write -> read. I will add them once we have write operations ready and enable the asserts